### PR TITLE
Leaderboard Setup project

### PR DESCRIPTION
**Pull request by JakariaJishan**
- Use callbacks and promises.
- Use proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.
- Use webpack to bundle JavaScript.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/i,
+        type: 'asset/resource',
+      },
+    ],
+  },
+
+  //   optimization: {
+  //     runtimeChunk: "single",
+  //   },
```
Comment (danmainah) [inline webpack.config.js]: - [x] @JakariaJishan it's not good practice to have commented code in your codebase kindly remove the commented code, also try and resolve the linter error in line 3 it's not recommended to disable it unless its the last option.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @JakariaJishan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: webpack.config.js (hunk 1 above)
Comment (JakariaJishan) [reply webpack.config.js]: I tried but could not resolve the linter error in line 3. so I am with the last option.

**Review by huseyinbiyik — APPROVED** [review]
Hi @JakariaJishan,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Wireframe implemented well 👍 
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝

## Optional suggestions
- Not sure about your linter issue but, it seems a simple one from my side. 
Screenshot 2023-02-07 at 10 45 40 PM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**